polaris/ocean/tests/inertial_gravity_wave/viz.py

```diff
@@ -0,0 +1,124 @@
+        config = self.config
+        resolutions = self.resolutions
+        nres = len(resolutions)
+
+        section = config['inertial_gravity_wave']
+        eta0 = section.getfloat('eta0')
+
+        fig, axes = plt.subplots(nrows=nres, ncols=3, figsize=(12, 2 * nres))
+        rmse = []
+        for i, res in enumerate(resolutions):
+            init = xr.open_dataset(f'init_{res}km.nc')
+            ds = xr.open_dataset(f'output_{res}km.nc')
+            exact = ExactSolution(init, config)
+
+            t0 = datetime.datetime.strptime(ds.xtime.values[0].decode(),
+                                            '%Y-%m-%d_%H:%M:%S')
+            tf = datetime.datetime.strptime(ds.xtime.values[-1].decode(),
+                                            '%Y-%m-%d_%H:%M:%S')
+            t = (tf - t0).total_seconds()
+            ssh_model = ds.ssh.values[-1, :]
+            rmse.append(np.sqrt(np.mean((ssh_model - exact.ssh(t).values)**2)))
+
+            # Comparison plots
+            ds['ssh_exact'] = exact.ssh(t)
+            ds['ssh_error'] = ssh_model - exact.ssh(t)
+            if i == 0:
+                error_range = np.max(np.abs(ds.ssh_error.values))
+
+            plot_horiz_field(ds, init, 'ssh', ax=axes[i, 0],
+                             cmap='cmo.balance', t_index=ds.sizes["Time"] - 1,
+                             vmin=-eta0, vmax=eta0)
+            plot_horiz_field(ds, init, 'ssh_exact', ax=axes[i, 1],
+                             cmap='cmo.balance',
+                             vmin=-eta0, vmax=eta0)
+            plot_horiz_field(ds, init, 'ssh_error', ax=axes[i, 2],
+                             cmap='cmo.balance',
+                             vmin=-error_range, vmax=error_range)
+
+        axes[0, 0].set_title('Numerical')
+        axes[0, 1].set_title('Exact')
+        axes[0, 2].set_title('Error')
+
+        pad = 5
+        for ax, res in zip(axes[:, 0], resolutions):
+            ax.annotate(f'{res}km', xy=(0, 0.5),
+                        xytext=(-ax.yaxis.labelpad - pad, 0),
+                        xycoords=ax.yaxis.label, textcoords='offset points',
+                        size='large', ha='right', va='center')
+
+        fig.savefig('comparison.png', bbox_inches='tight', pad_inches=0.1)
+
+        # Convergence polts
+        fig = plt.figure()
+        ax = fig.add_subplot(111)
+        p = np.polyfit(np.log10(resolutions), np.log10(rmse), 1)
+        conv = np.round(p[0], 3)
+        ax.loglog(resolutions, rmse, '-ok', label=f'numerical (order={conv})')
+
+        c = rmse[0] * 1.5 / resolutions[0]
+        order1 = c * np.power(resolutions, 1)
+        c = rmse[0] * 1.5 / resolutions[0]**2
+        order2 = c * np.power(resolutions, 2)
+
+        ax.loglog(resolutions, order1, '--k', label='first order', alpha=0.3)
+        ax.loglog(resolutions, order2, 'k', label='second order', alpha=0.3)
+        ax.set_xlabel('resolution (km)')
+        ax.set_ylabel('RMS error (m)')
```

Comment:
```suggestion
        ax.set_ylabel('RMS error (m)')
        ax.invert_xaxis()
```
Should we flip this for convention's sake? The other convergence plots in polaris use nCells so that the slopes decrease from left to right.